announcements/2020-Week-44/announcement.md

```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [Cert Manager v 2.3.1](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#231---2020-10-29) removes deprecated APIs and upgrades to upstream v1.0.3.
+2. [Kong v0.9.2](https://github.com/giantswarm/kong-app/blob/release-v0.9.x/CHANGELOG.md#092---2020-10-28) adds minReadySeconds to deployment template as an optional parameter 
+
```

Comment:
Waiting for @piontec EFK release https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#unreleased 

Some suggested text

improves reliability of upgrades. It enables PodDisruptionBudget by default, adds tests to ensure ..., and .

@giantswarm/app-squad-efk please help fill in :P